### PR TITLE
Add Firstyear

### LINKED ISSUE #24: New Mentor: Firstyear
### William ([@Firstyear](https://github.com/Firstyear/))
* **Pronouns**: he/him/they/them
* **Contact**: Twitter ([@Erstejahre] (https://twitter.com/Erstejahre)), Email (william at blackhats.net.au)
* **Spoken Languages**: English
* **Topics**: Beginners, C-FFI, Concurrency, Data Structures, Security, Identity Management aka Authentication
* **Additional Resources**: https://fy.blackhats.net.au/blog/html/2019/02/26/programming_lessons_and_methods.html
https://fy.blackhats.net.au/blog/html/pages/ldap_guide_part_1_foundations.html
https://fy.blackhats.net.au/blog/html/2019/04/13/using_rust_generics_to_enforce_db_record_state.html
https://fy.blackhats.net.au/blog/html/2019/07/16/cpu_atomics_and_orderings_explained.html

### PR DESCRIPTION
**Pull request by yaahc**
closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,18 @@ Work in progress - https://github.com/RustBeginners/awesome-rust-mentors/issues/
 * **Spoken Languages**: English
 * **Topics**: Beginners/the basics, developer tools, CLI, Rust–Swift/Objective-C interop
 
+### William ([@Firstyear](https://github.com/Firstyear/))
+* **Pronouns**: he/him/they/them
+* **Contact**: Twitter ([@Erstejahre](https://twitter.com/Erstejahre)), Email (william at blackhats.net.au)
```
Comment (yaahc) [inline README.md]: Did you want us to leave your email in the broken form here? For all other emails we've made it into a mailto link and would like to keep it consistent if it's okay with you. But I understand if you don't want people automatically scrapping your email from the readme.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,18 @@ Work in progress - https://github.com/RustBeginners/awesome-rust-mentors/issues/
...
+* **Spoken Languages**: English
+* **Topics**: Beginners, C-FFI, concurrency, data structures, security, identity management aka authentication, code review
+* **Additional Resources**:
+    * [Programming lessons and methods](https://fy.blackhats.net.au/blog/html/2019/02/26/programming_lessons_and_methods.html)
```
Comment (yaahc) [inline README.md]: Are these titles okay?

File: README.md [hunk 3]
```diff
@@ -64,6 +64,18 @@ Work in progress - https://github.com/RustBeginners/awesome-rust-mentors/issues/
 * **Spoken Languages**: English
 * **Topics**: Beginners/the basics, developer tools, CLI, Rust–Swift/Objective-C interop
 
+### William ([@Firstyear](https://github.com/Firstyear/))
+* **Pronouns**: he/him/they/them
+* **Contact**: Twitter ([@Erstejahre](https://twitter.com/Erstejahre)), Email (william at blackhats.net.au)
+* **Spoken Languages**: English
+* **Topics**: Beginners, C-FFI, concurrency, data structures, security, identity management aka authentication, code review
```
Comment (yaahc) [inline README.md]: I added code review here because you mentioned it in the tweet.

**Comment by yaahc** [comment]
@Firstyear

File: README.md (hunk 1 above)
Comment (Firstyear) [reply README.md]: You can make it a mailto link, I just do this out of habit :)

File: README.md (hunk 2 above)
Comment (Firstyear) [reply README.md]: Perfect!

File: README.md (hunk 3 above)
Comment (Firstyear) [reply README.md]: Maybe it would be better to have code review before all the other things in the list? Otherwise all good to add this thank you

Comment (yaahc) [reply README.md]: If you want code review first so it shall be!

**Comment by Firstyear** [comment]
10/10 commit to production

**Pull request merged**